actions/dependabot-auto-triage/README.md

```diff
@@ -0,0 +1,111 @@
+# Auto Dismiss Dependabot Alerts
+
+A GitHub composite action to automatically dismiss Dependabot alerts based on manifest paths using glob patterns.
+
+## Usage
+
+This action can be used in a workflow to automatically dismiss Dependabot alerts for specific manifest paths. This is particularly useful for repositories with dependencies that are not directly used in production or for which vulnerabilities may not be relevant.
+
+### Example Workflow
+
+Create a workflow file (e.g., `.github/workflows/auto-dismiss-dependabot-alerts.yml`) with the following content:
+
+```yaml
+name: Auto Dismiss Dependabot Alerts
+
+on:
+  # Run daily to dismiss new alerts
+  schedule:
+    - cron: "0 0 * * *"
+
+  # Allow manual triggering
+  workflow_dispatch:
+
+jobs:
+  auto-dismiss:
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+      contents: read
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      # Get GitHub App token with Dependabot alerts permissions
+      - name: Retrieve GitHub App secrets
+        id: get-secrets
+        uses: grafana/shared-workflows/actions/get-vault-secrets@get-vault-secrets-v1.1.0
+        with:
+          common_secrets: |
+            DEPENDABOT_AUTO_TRIAGE_APP_ID=dependabot-auto-triage:app-id
+            DEPENDABOT_AUTO_TRIAGE_APP_PRIVATE_KEY=dependabot-auto-triage:private-key
+
+      - name: Generate token
+        id: generate-token
+        uses: actions/create-github-app-token@3ff1caaa28b64c9cc276ce0a02e2ff584f3900c5 # v2.0.2
+        with:
+          app-id: ${{ env.DEPENDABOT_AUTO_TRIAGE_APP_ID }}
+          private-key: ${{ env.DEPENDABOT_AUTO_TRIAGE_APP_PRIVATE_KEY }}
+
+      # Use the token with the auto-triage action
+      - name: Auto Dismiss Dependabot Alerts
+        uses: ./.github/actions/auto-triage
```

Comment:
Please update the uses-value before merging 🙂

Reply:
Done!